src/main/webapp/app/shared/select/GeneSelect.tsx

```diff
@@ -39,9 +40,29 @@ const GeneSelect = (props: IGeneSelectProps) => {
     };
   };
 
+  const Option: React.FunctionComponent<any> = (optionProps: any) => {
+    const hasAliases = optionProps.data.geneAliases?.length > 0;
+    return (
+      <>
+        <components.Option {...optionProps}>
+          <div>{optionProps.data.label}</div>
+          {hasAliases && (
+            <div style={{ color: 'grey', fontSize: '0.9em' }}>
+              <span>Also known as </span>
+              <span>{optionProps.data.geneAliases.map(alias => alias.name).join(', ')}</span>
+            </div>
+          )}
+        </components.Option>
```

Comment:
Can we move this to the shared folder so the gene and drug dropdowns or any dropdown option has subtitle? I believe the style will be the same and they should use the same component.